verticaltabs.js

```diff
@@ -467,8 +467,10 @@ VerticalTabs.prototype = {
     };
 
     window.gBrowser._endRemoveTab = (aTab) => {
-      window.gBrowser._blurTab(aTab);
-      aTab.classList.add('tab-hidden');
+      if (aTab._endRemoveArgs) {
+        window.gBrowser._blurTab(aTab);
+        aTab.classList.add('tab-hidden');
+      }
```

Comment:
Do we want to be a little more like the [base implementation](https://dxr.mozilla.org/mozilla-central/source/browser/base/content/tabbrowser.xml#2821) of `_endRemoveTab`, and call `return;` instead?